kernel_gateway/gatewayapp.py

```diff
@@ -298,6 +300,15 @@ def keyfile_default(self):
     def client_ca_default(self):
         return os.getenv(self.client_ca_env)
 
+    ssl_version_env = 'KG_SSL_VERSION'
+    ssl_version_default_value = ssl.PROTOCOL_TLSv1_2
+    ssl_version = Integer(ssl_version_default_value, config=True, allow_none=True,
+                        help="""Sets the SSL version to use for the web socket connection. (KG_SSL_VERSION env var)""")
+    
+    @default('ssl_version')
+    def ssl_version_default(self):
+        return int(os.getenv(self.ssl_version_env, self.ssl_version_default_value))
```

Comment:
I think we might need to follow suit with how the cert-related attributes are handled - which essentially default to None.  This means you'll need to do a little extra processing in `ssl_version_default()` to convert the numeric string (if an ENV is present) to an int, otherwise return None.